fastlane/lib/fastlane/actions/docs/capture_android_screenshots.md

```diff
@@ -52,7 +52,25 @@ Ensure that the following permissions exist in your **src/debug/AndroidManifest.
 
 ##### Configuring your <a href="#ui-tests">UI Tests</a> for Screenshots
 
-1. Add `@ClassRule public static final LocaleTestRule localeTestRule = new LocaleTestRule();` to your tests class to handle automatic switching of locales
+1. Add `LocaleTestRule` to your tests class to handle automatic switching of locales. 
```

Comment:
*sigh Markdown

```suggestion
1. Add `LocaleTestRule` to your tests class to handle automatic switching of locales.  
```

Note the second space at the end of the line that will actually cause a line break.